TACC/Core-Styles
# fix(icon): hide vert overflow inside icon (from wrapped text)

## Overview

Hide vertical overflow inside icon (from wrapped text).

## Related

- fixes incomplete implementation of #86

## Changes

- added height to hide vertical overflow

## Testing

1. Create an icon with much text inside.

    ```
    <i class="icon icon-user">Lorem ipsum is really long text. And just like any text, it can wrap.</i>
    ```

2. Render the icon.
3. Verify _icon_ **is** visible.
4. Verify _text_ **not** visible.

## UI

![hotfix icon vert overflow](https://user-images.githubusercontent.com/62723358/207412948-ce7819d6-d76b-47ec-896d-d25de8e42416.png)
